common/changes/office-ui-fabric-react/breadcrumbA11y_2018-10-29-22-40.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Breadcrumb Example: distinct aria labels",
+      "type": "patch"
```

Comment:
I'm curious, should we mark this as `none` since it only impacts documentation?

Reply:
I was wondering about that. Yeah I think that makes sense!